R/utils.R

```diff
@@ -1,17 +1,11 @@
 getLocalTags <- function() {
+  shiny::addResourcePath("shinydiscon", system.file("src", package = "shinydisconnect"))
+  js <- htmltools::tags$head(htmltools::tags$script(src = "shinydiscon/js/shinydisconnect.js"))
   if (!isLocal()) {
-    return(NULL)
+    return(js)
```

Comment:
This should be done in the parent file, where this function gets called. The `getLocalTags()` is meant to only return HTML that is needed when running locally, so if `js` is returned in both cases, it should just be included in the level above.